fvm/evm/emulator/emulator.go

```diff
@@ -155,8 +155,8 @@ func (bl *BlockView) RunTransaction(
 	if err != nil {
 		return nil, err
 	}
-	res.TxHash = txHash
```

Comment:
Is the `TxHash` being set in another way now?

Reply:
it's assigned in the procedure itself

Reply:
Yeah its set inside the run method, it was a duplicated logic